exercises/03-sagas/instructions.md

```diff
@@ -232,30 +232,24 @@ In the saga add a new async method called `ProcessOrder` that you will call from
     }
 ```
 
-## Advanced exercises
 
-These advanced exercises are provided as additional challenges. They don't have such detailed guidance and we don't provide solutions for them. If you feel like it, you can do them during the workshop, in your hotel room, or at home. If you have questions, you can ask them during the workshop or using the Particular Software free support channel in Google Groups: https://groups.google.com/forum/#!forum/particularsoftware.
+## Advanced Exercise 3.5
 
-### Advanced exercise 1
+**Important: Before attempting the advanced exercises, please ensure you have followed [the instructions for preparing your machine for the advanced exercises](README.md#preparing-your-machine-for-the-advanced-exercises).**
 
-This exercise does not require existing NServiceBus skills, but you will probably need to read [the documentation](http://docs.particular.net/nservicebus/sagas/timeouts) to complete it.
+Sagas are excellent for coordinating a business process. In the current saga we're only able to execute the happy path of the business process, where everything works. Payment succeeds and arrives within a short time. But what if the `PaymentSucceededEvent` never arrives?
 
-Sagas are excellent for coordinating a business process. In the current saga we are only able to execute the happy path of the business process. The path where everything goes well. Payment succeeds and arrives within a short time. But what if the `PaymentSucceededEvent` never arrives?
+We can request a timeout in our saga so that, after a certain amount of time when nothing has happened, we can take action such as notifying the shipping or sales department that there may be a problem. A saga in the Finance bounded context would take care of this, but let's also take some action after the timeout elapses. Otherwise, this saga would live forever without anyone noticing the order processing failed in some way.
```

Comment:
There doesn't actually seem to be an exercise here. What do we expect the user to do here?